tests/conftest.py

```
@@ -452,6 +456,14 @@ def schedulable_nodes(nodes):
         and node.kubelet_ready
     ]
 
+    cpu_arch = py_config.get("cpu_arch")
+    if cpu_arch:
+        schedulable = [node for node in schedulable if node.labels.get(KUBERNETES_ARCH_LABEL) == cpu_arch]
+
+    arch_info = f" (filtered by arch {cpu_arch})" if cpu_arch else ""
+    LOGGER.info(f"Schedulable nodes{arch_info}: {[node.name for node in schedulable]}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)